portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,88 @@
 #content {
   margin-left: auto;
   margin-right: auto;
+  margin-top: 20px;
   width: 650px;
+  padding: 20px;
+  border: 5px solid #333;
+  border-radius: 6px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+.body{
+    background:gainsboro;
+    color:black;
+    margin-left: auto;
+}
+
+.resume{
```

Comment:
Are you going to add the resume link to the main page in a future PR?

Reply:
A future PR, I'm trying out different looks and I left that there to test

Reply:
I removed it. I'll add it back later in the next PR.